src/main/java/gov/nysenate/openleg/controller/api/admin/AdminApiEmailCtrl.java

```diff
@@ -0,0 +1,53 @@
+package gov.nysenate.openleg.controller.api.admin;
+
+import gov.nysenate.openleg.controller.api.base.BaseCtrl;
+import gov.nysenate.openleg.model.auth.ApiUserSubscriptionType;
+import gov.nysenate.openleg.service.mail.apiuser.ApiUserBatchEmailServiceImpl;
+import gov.nysenate.openleg.service.mail.apiuser.ApiUserEmailRequest;
+import gov.nysenate.openleg.service.mail.apiuser.ApiUserMessage;
+import org.apache.shiro.SecurityUtils;
+import org.apache.shiro.authz.annotation.RequiresPermissions;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static gov.nysenate.openleg.controller.api.base.BaseCtrl.BASE_ADMIN_API_PATH;
+
+@RestController
+@RequestMapping(BASE_ADMIN_API_PATH + "/email")
+public class AdminApiEmailCtrl extends BaseCtrl {
+
+    @Autowired
+    ApiUserBatchEmailServiceImpl apiUserBatchEmailService;
+
+
+    @RequiresPermissions("admin:email:post")
```

Comment:
The Ctrl files could use some javadocs.  See other Ctrl files for examples of how we document our api controllers.